styles.css

```diff
@@ -55,11 +55,6 @@ nav a {
   padding-right: 10rem;
 }
 
```

Comment:
- [ ] [REQUIRED] Good job so far! Please remove necessary `space` from right side of your web page and implement `border-radius` with required value to battom part of the section 🙏🏼 

![image](https://github.com/Tshepo45/Mobile-First/assets/63915024/c4b54715-f9d6-4807-8014-5ffa25a066f3)